### ISSUE TITLE
Implementation details

### ISSUE THREAD

**Issue by clelouch** [backseason/DFI#1]
Thanks for your amazing work.
After reading your paper and code, I have few question about some implementation details.
1. I notice that you set the bias to false in almost all convolution layers except the prediction layers. Can you tell me the reason?

2. Instead of using Batchnorm (BN), you use Groupnorm (GN) . Did you compare these two methods and how much improvement GN brings?

3. While in most blocks relu layer is applied after the GN layer, I notice that in some blocks you don't use the relu layer.  For example:
![image](https://user-images.githubusercontent.com/33519394/93784538-b6f7b080-fc5f-11ea-8bfc-40586ec2d71d.png)
I am curious about how to judge whether a relu layer is necessary.

**Comment by backseason** [comment]
Thanks for your attention to our work.
1. It is followed by the implementation of ResNet and other popular CNNs. You may do some google as I forget what's the paper that it comes from.
2. The model is trained with `batch_size=1`, where BN is not suitable as it requires large batch size. So we adopt GN instead.
3. I actually did not pay much attention to that... It's not carefully designed as I always think it brings minor effects. I usually place ReLU after each Conv-BN(GN).

**Comment by clelouch** [comment]
Many thanks for your reply.
